apps/mark/frontend/test/helpers/mock_api_client.tsx

```diff
@@ -31,6 +35,16 @@ import { mockMachineConfig } from './mock_machine_config';
 import { initialElectionState } from '../../src/app_root';
 import { ApiProvider } from '../../src/api_provider';
 
+// the below is copied from libs/printing to avoid importing a backend package
+export const MOCK_PRINTER_CONFIG: PrinterConfig = {
+  label: 'HP LaserJet Pro M404n',
+  vendorId: 1008,
+  productId: 49450,
+  baseDeviceUri: 'usb://HP/LaserJet%20Pro%20M404-M405',
```

Comment:
VID/PID are typically written in hex notation. Searching the internet for the hex values yields better results.
```suggestion
  vendorId: 0x03f0,
  productId: 0xc12a,
```
I realize this was likely just copied from the JSON file where we can't use hex notation. I don't think there's actually a good reason to have `configs.json` as JSON. We should probably just make it a TS file.

Reply:
I think part of the reason is that it used to exist in `vxsuite-complete-system` and be read in as a config file for `kiosk-browser`. Now it doesn't need to be. I do like how it's co-located with the `.ppd` files that it references, outside of the `src` directory. Going to leave as-is for now to move on to next PR.